app/retail/templates/emails/shared_template_head.html

```diff
@@ -14,10 +14,6 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
-{% load i18n static compress %}
-{% compress css inline email_head %}
-  <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/gitcoin.scss" %} />
-  <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/town_square.scss" %} />
-  <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/activity_stream.scss" %} />
-{% endcompress %}
```

Comment:
Hey @chibie, I've just tried this change out on the stable branch, and we are using these styles for a few of the emails. Could you have a look at #8553 and reintroduce the inline css as it's done there 👍 😄 Thank you!!

Reply:
@gdixon

`new_bounty_daily` will be purged in favour of `new_bounty` which has the current requirement for the **daily digest** email (activities, quests and stuff were taken out)

can you checkout staging to this pr branch so we regress there?